src/libraries/System.Private.DataContractSerialization/src/ILLink/ILLink.Suppressions.xml

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<linker>
+  <assembly fullname="System.Private.DataContractSerialization, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
```

Comment:
I don't understand this. I thought you fixed this assembly in [https://github.com/dotnet/runtime/pull/50619](https://github.com/dotnet/runtime/pull/50619). Why are we getting these now?

Reply:
I think this is because the wrong configuration of System.Private.DataContractSerialization is getting picked. One more reason why not to go with this approach I guess.